docs/content/documentation/content/image-processing/index.md

```diff
@@ -106,7 +106,7 @@ The source for all examples is this 300 pixel × 380 pixel image:
 
 ## Using `resize_image` in markdown via shortcodes
 
-`resize_image` is a built-in Tera global function (see the [templates](@/documentation/templates/_index.md) chapter),
```

Comment:
This is a bit tricky. The current doc says that `resize_image()` is a built Tera function but https://tera.netlify.app/docs/#built-in-functions has no `resize_image()` since it's a built-in Zola function. Ref - https://www.getzola.org/documentation/templates/overview/#built-in-global-functions

Reply:
Yeah it's a bit hard to parse. It's a built-in (by Zola) Tera function

Reply:
Okay. So how about "`resize_image` is a Zola built-in global function written in Tera template engine (see the [templates](@/documentation/templates/_index.md) chapter),"?

Reply:
`resize_image` is a Zola built-in Tera function (see the [templates](@/documentation/templates/_index.md) chapter)

?

Reply:
Sure!